appholder/src/main/java/com/android/mdl/app/MainActivity.kt

```diff
@@ -101,13 +99,18 @@ class MainActivity : AppCompatActivity() {
         }
         logInfo("uri: $mdocUri")
 
-        val originInfos = ArrayList<OriginInfo>()
+        val originInfos = ArrayList<com.android.identity.mdoc.origininfo.OriginInfo>()
```

Comment:
The `com.android.identity.mdoc.origininfo.` part seems unnecessary since you import this above? There's a bunch of places where this happen, I guess a side-effect of refactoring? Either way, would be nice to clean it up before landing.